ledger/acctupdates_test.go

```diff
@@ -1352,6 +1352,191 @@ func TestBoxNamesByAppIDs(t *testing.T) {
 	}
 }
 
+func TestBoxCache(t *testing.T) {
+	partitiontest.PartitionTest(t)
+
+	initialBlocksCount := 1
+	accts := make(map[basics.Address]basics.AccountData)
+
+	protoParams := config.Consensus[protocol.ConsensusCurrentVersion]
+	protoParams.MaxBalLookback = 8
+	protoParams.SeedLookback = 1
+	protoParams.SeedRefreshInterval = 1
```

Comment:
Why are you setting these? I don't think they do what you think.  See the most recent merge to feature/avm-box, where I had to change this.  You (probably) want conf.MaxAcctLookback.